sw/device/lib/dif/dif_clkmgr.c

```diff
@@ -88,8 +88,19 @@ dif_result_t dif_clkmgr_jitter_set_enabled(const dif_clkmgr_t *clkmgr,
     default:
       return kDifBadArg;
   }
-  mmio_region_write32(clkmgr->base_addr, CLKMGR_JITTER_ENABLE_REG_OFFSET,
-                      new_jitter_enable_val);
+
```

Comment:
I don't quite agree with this behavior: I would rather model it as
  new_jitter_enable_val = kMultiBitBool4True;
  mmio_region_write32(clkmgr->base_addr, CLKMGR_JITTER_ENABLE_REG_OFFSET,
                       new_jitter_enable_val);

Also the writes do occur, though not with the expected value, so I would not return kDifLocked. The real issue is that this is not a true RW CSR.